cmd/zq/README.md

```diff
@@ -20,21 +20,21 @@ data, which is used in the examples in the
 [query language documentation](../../zql/docs/README.md).
 
 To cut the columns of a Zeek "conn" log like `zeek-cut`, and output to the
- terminal, use `cut`:
+ terminal, use [`cut`](../../zql/docs/processors/README.md#cut):
 
 ```
-zq -t "* | cut ts,id.orig_h,id.orig_p" conn.log
+zq -z "* | cut ts,id.orig_h,id.orig_p" conn.log
 ```
 
-The `-t` tells `zq` to use [TZNG](../../zng/docs/spec.md#4-zng-text-format-tzng)
+The `-z` tells `zq` to use human-readable text [ZSON](../../zng/docs/zson.md)
```

Comment:
While they certainly don't mean the same thing, in this context "text" feels redundant after "human-readable".
```suggestion
The `-z` tells `zq` to use human-readable [ZSON](../../zng/docs/zson.md)
```